README.md

```diff
@@ -1,17 +1,198 @@
-# Plastic SCM plugin for Jenkins
+<h1 align="center">
+    <img src="doc/img/logo-plasticscm.svg" alt="Plastic SCM Logo" width="450" />
```

Comment:
I recommend using the GitHub Markdown syntax. I am not 100% sure it will be rendered well on plugins.jenkins.io, but we can take a look. Anyway, there is a story for supporting plugin logos in Jenkins marketplaces: https://issues.jenkins-ci.org/browse/WEBSITE-645

CC @zbynek

Reply:
Good point. I took this other README as reference: https://github.com/toolmantim/release-drafter/blob/master/README.md

I thought the logo & description looked neat centered, so that's why I switched to plain HTML... Since Markdown should be compatible, I hope nothing breaks in plugins.jenkins.io. However, if something goes wrong I'll fall back to plain GHF Markdown 😊